tutorials/gui_config.md

```diff
@@ -15,10 +15,34 @@ There are a few places where the GUI configuration can come from:
 2. A `<gui>` element inside an SDF file
 3. The default configuration file at `$HOME/.ignition/gazebo/gui.config` \*
 
-Each of the items above takes precedence over the ones below it. For example,
-if a user chooses a `--gui-config`, the SDF's `<gui>` element is ignored. And
-the default configuration file is only loaded if no configuration is passed
-through the command line or the SDF file.
+Each of the items above takes precedence over the ones below it by default.
+For example, if a user chooses a `--gui-config`, the SDF's `<gui>` element is
+ignored. And the default configuration file is only loaded if no configuration
+is passed through the command line or the SDF file.
+
+The default behaviour can be overridden using the `--gui-config-options`
+command line argument, which offers options:
+
+* `force`: Force the use of the GUI config file, ignoring the SDF file. If a
```

Comment:
How about making `force` override `--gui-config` as well? Specifying both `--gui-config-options force` and `--gui-config` is a bit odd, and probably will never be done. However, it seems easier and cleaner to  remember that `force` is a complete `force` without a caveat.

Reply:
I'm not sure I understand the proposal. Are you saying that `--gui-config <path_to_file>` should imply `force`? I believe that's already in the current proposal. If `--gui-config` is provided and `--gui-config-option` is empty, it works like a `force`.

See these lines on the table below:

```
--gui-config     | SDF `<gui>`  | --gui-config-option      | Result
---------------- | ------------ | ------------------------ | ------
provided         | no plugins   | empty / force / prepend  | Load plugins from `--gui-config`
provided         | has plugins  | empty / force            | Load plugins from `--gui-config`
```